draft-ietf-trans-rfc6962-bis.md

```diff
@@ -929,18 +932,16 @@ which encapsulates a `SignedCertificateTimestampDataV2` structure:
         LogID log_id;
         uint64 timestamp;
         Extension sct_extensions<0..2^16-1>;
-        digitally-signed struct {
-            TransItem timestamped_entry;
-        } signature;
+        SignatureScheme algorithm;
+        opaque signature<0..2^16-1>;
     } SignedCertificateTimestampDataV2;
 ~~~~~~~~~~~
 
 `log_id` is this log's unique ID, encoded in an opaque vector as described in
 {{log_id}}.
 
-`timestamp` is equal to the timestamp from the
-`TimestampedCertificateEntryDataV2` structure encapsulated in the
-`timestamped_entry`.
+`timestamp` is equal to the timestamp from the underlying
```

Comment:
We use "corresponding" in the previous section to direct the reader to this section's definition of "sct_extensions", so I suggest you do: "s/underlying/corresponding/"